File: genomic/genomics_api/models/dtos/GeneticVariantDtos/VariantInDTO.py
class VariantInDTO:
    def __init__(self, gene_id, chromosome, position, reference_base, alternate_base, impact):
        self.gene_id = gene_id
        self.chromosome = chromosome
        self.position = position
        self.reference_base = reference_base
        self.alternate_base = alternate_base
        self.impact = impact

File: authentication/src/main/java/com/genosentinel/authentication/models/dto/variantDTO/VariantInDTO.java
package com.genosentinel.authentication.models.dto.variantDTO;

import lombok.Data;

@Data
public class VariantInDTO {
    private Long gene_id;
    private String chromosome;
    private Long position;
    private String reference_base;
    private String alternate_base;
    private Impact impact;

    public enum Impact {
        Missense,
        Frameshift,
        Nonsense,
        Silent,
        Splice,
        Other
    }

}
